### PR TITLE
Add dev log for v24.04

### PR DESCRIPTION
**Pull request by AaronVanGeffen**

### CODE REVIEW

File: _posts/2024-04-07-openloco-v24.04.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Error messages appearing below the window after cloning a vehicle (#2403)
+
+After a recent change to the game command structure for cloning vehicles, a mistake slipped in:
+the error state was not being checked properly anymore! First-time contributor @Sebazzz noticed what was
+going wrong, and quickly came up with a patch for it. Thanks!
+
+## Crash due to the station limit (#2396)
+
+Our shiny new station limit checks, which we presented in v24.03, introduced a classic of-by-one error.
```
Comment (LeftofZen) [inline _posts/2024-04-07-openloco-v24.04.md:36]: `off-by-one`

File: _posts/2024-04-07-openloco-v24.04.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+title:      "OpenLoco version 24.04"
+tagline:    "Release blog post"
+author:     "Aaron van Geffen"
+date:       2024-04-07 17:00:00 +0200
+layout:     single
+classes:    wide
+categories: news
+---
+
+OpenLoco v24.04 is out! We're releasing this one a little earlier than you might have expected, little over a week after v24.03.
```
Comment (LeftofZen) [inline _posts/2024-04-07-openloco-v24.04.md:11]: I'd write this as
`have expected; a little over a week`

File: _posts/2024-04-07-openloco-v24.04.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+However, besides quite a bit of behind the scenes work, many bugfixes made it in. We think that warrants a new one!
```
Comment (LeftofZen) [inline _posts/2024-04-07-openloco-v24.04.md:12]: technically this should be `bug fixes` but I don't mind either way

File: _posts/2024-04-07-openloco-v24.04.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+After a recent change to the game command structure for cloning vehicles, a mistake slipped in:
+the error state was not being checked properly anymore! First-time contributor @Sebazzz noticed what was
+going wrong, and quickly came up with a patch for it. Thanks!
+
+## Crash due to the station limit (#2396)
+
+Our shiny new station limit checks, which we presented in v24.03, introduced a classic of-by-one error.
+In spite of the more flexible limit checks, stations can still only occupy 80 tiles. However, our implementation
+checked for 80 tiles inclusive, which meant players were allowed to place an 81st tile... Leading to surreptitious
```
[the remark this answers is not in the document]
Comment (AaronVanGeffen) [reply _posts/2024-04-07-openloco-v24.04.md:38]: I was a bit surprised to read this, but apparently the style [varies widely](https://en.wikipedia.org/wiki/Ellipsis#British_English). I'll remove the ellipses entirely; apparently the younger generation generally considers it passive aggressive 😄

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**